bitnami/dataplatform-bp2/README.md

```diff
@@ -391,7 +393,7 @@ $ helm install my-release -f values.yaml bitnami/dataplatform-bp2
 
 In the default deployment, the helm chart deploys the data platform with [Metrics Emitter](https://hub.docker.com/r/bitnami/dataplatform-emitter) and [Prometheus Exporter](https://hub.docker.com/r/bitnami/dataplatform-exporter) which emit the health metrics of the data platform which can be integrated with your observability solution.
 
-In case you need to deploy the data platform with Tanzu Observability Framework for all the applications (Kafka/Spark/Elasticsearch/Logstash) in the data platform, you can specify the 'enabled' parameter using the `--set <component>.metrics.enabled=true` argument to `helm install`. For Example,
+To deploy the data platform with Tanzu Observability Framework with the Wavefront Collector for all the applications (Kafka/Spark/Elasticsearch/Logstash) in the data platform, you can specify the 'enabled' parameter using the `--set <component>.metrics.enabled=true` argument to `helm install`. For Example,
```

Comment:
Can we have 2 sections here for clarity? 
1. With Tanzu Observability Deployment  
- Mention the auto-discovery default configuration
- seperate heading in case they want to disable auto-discovery

2. Using an existing Tanzu Observability Deployment
- Mention the auto-discovery default configuration
- seperate heading in case they want to disable auto-discovery